frontend/src/Login.jsx

```diff
@@ -13,21 +38,35 @@ const Login = () => {
       <form className="flex flex-col bg-indigo-50 p-8 rounded-md shadow-xl/40 shadow-slate-900 w-150 m-auto">
         <h3 className="font-bold text-3xl p-3 text-center"> Welcome Back</h3>
         <InputBox
-          placeholder={"Username*"}
-          label={"Username"}
-          name={"username"}
+          placeholder="Username*"
+          label="Username"
+          name="username"
+          value={formData.username}
+          handleFormChange={handleFormChange}
         />
         <InputBox
           placeholder={"Password*"}
```

Comment:
Same as above, remove `{}` for simple strings. Also the asterisks at the end of the strings.